api/v1/hardening_types.go

```diff
@@ -41,5 +41,6 @@ type CBContainersEnforcerSpec struct {
 	// +kubebuilder:default:=<>
 	Probes CBContainersHTTPProbesSpec `json:"probes,omitempty"`
 	// +kubebuilder:default:=5
-	WebhookTimeoutSeconds int32 `json:"webhookTimeoutSeconds,omitempty"`
+	WebhookTimeoutSeconds  int32 `json:"webhookTimeoutSeconds,omitempty"`
+	EnableEnforcingWebhook bool  `json:"enableEnforcingWebhook,omitempty"`
```

Comment:
cool!

Reply:
maybe change the name from EnableEnforcingWebhook to EnableEnforcementFeature? 
(I think in the CR it will be better to enable/disable feature than a component and ofc the feature can contains multiple component)

Reply:
In addition I think it will be good to explicit set the default value to false in octarine-operator/controllers/basic_components_defaults.go